base/logger_file.go

```diff
@@ -13,9 +14,12 @@ import (
 var (
 	ErrInvalidLogFilePath = errors.New("invalid log file path")
 
-	maxAgeLimit             = 9999 // days
-	defaultMaxSize          = 100  // 100 MB
-	defaultMaxAgeMultiplier = 2    // e.g. 90 minimum == 180 default maxAge
+	maxAgeLimit                            = 9999 // days
+	defaultMaxSize                         = 100  // 100 MB
+	defaultMaxAgeMultiplier                = 2    // e.g. 90 minimum == 180 default maxAge
+	defaultCumulativeMaxSizeBeforeDeletion = 300  // 300 MB
```

Comment:
Compare this to the 2.0 defaults (max age + max num) to evaluate whether this is sufficient.